rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -460,6 +487,447 @@ MasterGraph::timing()
     return t;
 }
 
+#define CHECK_CL_CALL_RET(x) { cl_int ret; ret = x; if( ret != CL_SUCCESS) THROW("ocl call failed "+STR(#x)+" error "+TOSTR(ret)) }
+
+MasterGraph::Status
+MasterGraph::to_tensor(void *out_ptr, RocalTensorlayout format, float multiplier0, float multiplier1,
+                             float multiplier2, float offset0, float offset1, float offset2, bool reverse_channels, RocalTensorDataType output_data_type, RocalOutputMemType output_mem_type)
+{
+    if(no_more_processed_data())
+        return MasterGraph::Status::NO_MORE_DATA;
+    
+    if(_output_tensor_list.size() != 1)
+        THROW("Cannot copy, Multiple output tensors present in the list")
+    
+    if(_output_tensor_list[0]->info().data_type() != RocalTensorDataType::UINT8)
+        THROW("The output tensor is not of UINT8 type")
+
+    if (_output_tensor_list[0]->info().color_format() == RocalColorFormat::RGB_PLANAR)
+        return MasterGraph::copy_out_tensor_planar(out_ptr, format,multiplier0, multiplier1, multiplier2, offset0, offset1, offset2, reverse_channels, output_data_type);
+
+    _convert_time.start();
+    // Copies to the output context given by the user
+    auto dims = _output_tensor_list[0]->info().dims();
```

Comment:
please use another pointer to point to output_tensor_list[0]->info() and use that everywhere.